### PR TITLE
ovirt_vm: fix template search

### PR DESCRIPTION
**Pull request by mnecas**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1882451

### CODE REVIEW

File: plugins/modules/ovirt_vm.py [hunk 1]
```diff
@@ -1381,6 +1381,14 @@ def __get_template_with_version(self):
                 template = templates[0]
                 if self.param('template') is not None and self.param('template') != template.name:
                     raise ValueError("You can not change template of the Virtual Machine.")
+            else:
+                raise ValueError(
+                    "Template with name '%s' and version '%s' in data center '%s' was not found'" % (
```
Comment (kobihk) [inline plugins/modules/ovirt_vm.py:1386]: I think you have a redundant apostrophe in:
found'"

**Review by kobihk — APPROVED** [review]

**Review by mwperina — APPROVED** [review]
+1

**Comment by arachmani** [comment]
@mnecas I've just tested it and HE deployment failed with the following error:

The full traceback is:
Traceback (most recent call last):
  File "/tmp/ansible_ovirt_vm_payload_w6l93ge0/ansible_ovirt_vm_payload.zip/ansible_collections/ovirt/ovirt/plugins/modules/ovirt_vm.py", line 2581, in main
  File "/tmp/ansible_ovirt_vm_payload_w6l93ge0/ansible_ovirt_vm_payload.zip/ansible_collections/ovirt/ovirt/plugins/module_utils/ovirt.py", line 600, in create
    new_entity = self.build_entity()
  File "/tmp/ansible_ovirt_vm_payload_w6l93ge0/ansible_ovirt_vm_payload.zip/ansible_collections/ovirt/ovirt/plugins/modules/ovirt_vm.py", line 1454, in build_entity
  File "/tmp/ansible_ovirt_vm_payload_w6l93ge0/ansible_ovirt_vm_payload.zip/ansible_collections/ovirt/ovirt/plugins/modules/ovirt_vm.py", line 1389, in __get_template_with_version
UnboundLocalError: local variable 'data_center' referenced before assignment

**Review by arachmani — APPROVED** [review]
LGTM

**Pull request merged**